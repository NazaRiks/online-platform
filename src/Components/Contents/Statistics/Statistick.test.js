import { it, assert,describe, test, expect, mount } from 'vitest';
test('Statistic', () => {
    const data = { foo: new Set(['bar', 'snapshot']) }
    expect(data).toMatchSnapshot({ foo: expect.any(Set) })
})
describe('MyStatisticsPage Component', () => {
    test('renders without crashing', () => {
        const div = document.createElement('div');
        mount(MyStatisticsPage, div);
        expect(div.innerHTML).toMatchSnapshot();
    });
});