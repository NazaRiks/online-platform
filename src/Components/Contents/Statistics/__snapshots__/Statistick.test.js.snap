// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Statistic 1`] = `
{
  "foo": Any<Set>,
}
`;

exports[`forum list items 1`] = `
{
  "foo": Any<Set>,
}
`;
